testcases/OpTestHMIHandling.py

```diff
@@ -76,6 +76,20 @@ def setUp(self):
         self.cv_SYSTEM.goto_state(OpSystemState.OS)
         self.clearGardEntries()
         self.cv_HOST.host_enable_all_cores(console=1)
+        self.cpu = ''.join(self.cv_HOST.host_run_command("grep '^cpu' /proc/cpuinfo |uniq|sed -e 's/^.*: //;s/[,]* .*//;'"))
+        if self.cpu in ["POWER9"]:
+            self.revision = ''.join(self.cv_HOST.host_run_command("grep '^revision' /proc/cpuinfo |uniq|sed -e 's/^.*: //;s/ (.*)//;'", console=1))
+            if not self.revision in ["2.2"]:
+                log.debug("Skipping HMIHandling NOT supported on CPU={} Revision={}"
```

Comment:
I was very confused until I figured there's probably meant to be a comma after "Skipping"? :)

Reply:
> comma after "Skipping

Fixed.